src/brc20-bridge/src/memory.rs

```diff
@@ -0,0 +1,10 @@
+//! Memory ids for bridge
+//!
+//! DO NOT USE ANY MEMORY ID BELOW 10, since used by the sdk
```

Comment:
I would not use anything below 100 to leave more IDs for future upgrades of the SDK